app/components/billing-status-display/discount-timer-badge.hbs

```diff
@@ -1,16 +1,16 @@
 <LinkTo
   @route="pay"
   class="flex bg-gradient-to-b from-gray-600 to-gray-900 border border-yellow-600 rounded shadow-sm
-    {{if (eq @size 'small') 'px-1.5 py-1'}}
-    {{if (eq @size 'large') 'px-2 py-1.5'}}"
+    {{if (eq @size 'small') 'px-1.5 py-1 mr-3 text-xs'}}
```

Comment:
Doesn't make sense that we've have to set `mr-3` here - isn't this something that the parent can pass in?